app.js

```diff
@@ -1,6 +1,11 @@
 const express = require("express");
 const { getTopics } = require("./controllers/topics.controllers");
 const app = express();
+const endpoints = require("./endpoints.json");
+
+app.get("/api", (request, response) =>{
+    response.status(200).send({endpoints});
+})
```

Comment:
🌟 
Really nice job getting your head around this! I hope taking a break helped 👍 If you want to refactor after this week is done, consider moving this callback function into it's own controller function file, to make it more consistent with your other endpoints.

Reply:
I wanted to do it with the controller initially but I said I want to finish the tasks...I guess speed does involve a loss of quality at my current level